frontend/src/components/mine/Summary/MineSummary.js

```diff
@@ -147,9 +125,16 @@ export const MineSummary = (props) => {
               </Col>
             </Row>
             <Row gutter={16} type="flex">
-              {groupedPermits.map((permits) =>
-                renderSummaryPermit(permits[0], props.partyRelationships.filter(isActive))
-              )}
+              {props.mine.mine_permit
+                .filter((permit) => permit.permit_no.toUpperCase().charAt(1) !== "X")
+                .map((permit) =>
+                  renderSummaryPermit(permit, props.partyRelationships.filter(isActive))
+                )}
+              {props.mine.mine_permit
+                .filter((permit) => permit.permit_no.toUpperCase().charAt(1) === "X")
+                .map((permit) =>
+                  renderSummaryPermit(permit, props.partyRelationships.filter(isActive))
+                )}
```

Comment:
Same question here. Also, it seems like it might be useful to name the filter function(s). I know from previous discussions that this is separating them into exploratory permits, but that may not be clear to a future reader of this code (including me, if/when I forget)

Reply:
Do agree. I'll rename and add comments for clarity.

Reply:
Ended up moving this to the API and removing this logic from the front-end entirely.